medical_compliance/medical_compliance/api/resources.py

```diff
@@ -14,9 +17,39 @@ class Meta:
         authorization = Authorization()
 
 
-# class MeasurementNotificationResource(ModelResource):
-#     class Meta:
-#         queryset = MeasurementNotification.objects.all()
-#         resource_name = 'measurement-notifications'
-#         authentication = Authentication()
-#         authorization = Authorization()
+class WeightMeasurementResource(ModelResource):
+    class Meta:
+        queryset = WeightMeasurement.objects.all().order_by('-timestamp')
+        resource_name = 'weight-measurements'
+        paginator_class = Paginator
+        authentication = Authentication()
+        authorization = Authorization()
+
+    def prepend_urls(self):
+        return [
+            url(r"^(?P<resource_name>%s)/last_values%s$" % (self._meta.resource_name, trailing_slash()), self.wrap_view('get_last_values'), name="api_last_values"),
+        ]
+
+    def get_last_values(self, request, **kwargs):
+        self.method_check(request, allowed=['get'])
+        self.is_authenticated(request)
+        self.throttle_check(request)
+
+        lastWeightMeasurements = WeightMeasurement.objects.all().order_by('-timestamp')[:5]
+        amount = []
+        for measurement in lastWeightMeasurements:
+            amount = [measurement.value] + amount
+
+        status = "ok"
```

Comment:
I know. We also need to consider where we do the logic to analyze the measures and generate notifications - probably it would be best to broadcast to all the tasks that analyze weights